contracts/prices/primitives/KyberPriceFeed.sol

```diff
@@ -235,27 +234,23 @@ contract KyberPriceFeed is IPriceSource, DSMath {
             return (0, false);
         }
 
-        uint256 askRate = 10 ** (KYBER_PRECISION * 2) / bidRateOfReversePair;
+        uint256 askRate = (10 ** KYBER_PRECISION.mul(2)).div(bidRateOfReversePair);
```

Comment:
Can you confirm that this multiplies before exponentiating? If not, need `(KYBER_PRECISION.mul(2))`.